### PR TITLE
Regenerate controller using `code-generator` commit 6e2ffbc

### PR DESCRIPTION
**Pull request by akartsky**
Code generated from code-generator commit [6e2ffbc](https://github.com/aws-controllers-k8s/code-generator/commit/6e2ffbc3b16a30ac59be6719918c601c2c864064)

updated aws-go-sdk to v1.44.93

### CODE REVIEW

File: apis/v1alpha1/ack-generate-metadata.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ack_generate_info:
-  build_date: "2022-09-06T12:47:50Z"
-  build_hash: 585f06bbd6d4cc1b738acb85901e7a009bf452c7
-  go_version: go1.17.5
-  version: v0.20.0
+  build_date: "2022-09-20T05:27:13Z"
+  build_hash: 1da44f4be322eea156ed23a86e33c29da5cede9c
+  go_version: go1.19.1
+  version: v0.20.1-2-g1da44f4-dirty
```
Comment (A-Hilaly) [inline apis/v1alpha1/ack-generate-metadata.yaml:5]: Please update make the upgrade of runtime and code generator to `v0.20.1` in a seperate PR. The current runtime still 2 versions behijnd

Comment (akartsky) [reply apis/v1alpha1/ack-generate-metadata.yaml]: thanks done

**Review by surajkota — APPROVED** [review]
/approve
/lgtm

**Pull request merged**